nextflow.config

```diff
@@ -1,8 +1,78 @@
-singularity.enabled = false
-singularity.autoMounts = false
+profiles {
 
-docker.enabled = true
-process.container = 'haniffalab/webatlas-pipeline:0.3.2'
+  conda {
+    executor = 'local'
+    singularity.enabled = false
+    docker.enabled = false
+    conda.enabled = true
+  }
+
+  docker {
+    executor = 'local'
+    singularity.enabled = false
+    singularity.autoMounts = false
+
+    docker.enabled = true
+    process.container = 'haniffalab/webatlas-pipeline:0.3.2'
+  }
+
+  sanger_lsf {
+    singularity.enabled = true
+    singularity.autoMounts = false
+    process.container = 'haniffalab/webatlas-pipeline:0.3.2'
+
+    docker.enabled = false
+    singularity {
+        enabled = true
+        autoMounts = false
+        runOptions = '--bind /lustre --bind /nfs --cleanenv --home /tmp'
+    }
```

Comment:
You are duplicating the `singularity` config block. I think the first two lines of the sanger profile should be removed.